chain/mainnet/mezo_31612-1/genesis.json

```diff
@@ -1 +1,296 @@
-[]
+{
+  "app_name": "mezod",
+  "app_version": "1.0.0-rc2",
+  "genesis_time": "2025-04-14T08:50:53.767481Z",
+  "chain_id": "mezo_31612-1",
+  "initial_height": 1,
+  "app_hash": null,
+  "app_state": {
+    "auth": {
+      "params": {
+        "max_memo_characters": "256",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo188vx087k295fxdlx7f4wkznmghajnf0maysxpe",
+            "pub_key": null,
+            "account_number": "0",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo1qcjuy52698gf7xaymfdg00guh3nnse6chm0s2t",
+            "pub_key": null,
+            "account_number": "1",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo14ugypmyfn2whx4dhsw4jwflk2x5fx2vkvmjrzk",
+            "pub_key": null,
+            "account_number": "2",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo1lgrfdvcj38gwcwr0qv4ytdfcygc0mg6slsj3yt",
+            "pub_key": null,
+            "account_number": "3",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        }
+      ]
+    },
+    "authz": {
+      "authorization": []
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
```

Comment:
Let's double check the buckets and amounts. What we have in the PR description does not align with the order in the `balances` array. Bucket 1 is switched with bucket 2.

`mezo1qcjuy52698gf7xaymfdg00guh3nnse6chm0s2t` -> `0x0625C2515A29D09f1Ba4Da5a87BD1cbc67386758`

`mezo188vx087k295fxdlx7f4wkznmghajnf0maysxpe` -> `0x39D8679fd651689337e6f26aEB0a7B45fb29a5fb`

`mezo14ugypmyfn2whx4dhsw4jwflk2x5fx2vkvmjrzk` -> `0xAF1040eC899A9D7355B783aB2727F651a8932996`

`mezo1lgrfdvcj38gwcwr0qv4ytdfcygc0mg6slsj3yt` -> `0xfA0696b31289D0ec386f032A45B5382230Fda350`

Reply:
well, they are not in the same order, but the allocation matches the on in the description, this is what I've  noted when I checked earlier, which match the description:
```
mezo1qcjuy52698gf7xaymfdg00guh3nnse6chm0s2t -> 0x0625c2515a29d09f1ba4da5a87bd1cbc67386758 = 200000000000000000000000000
mezo188vx087k295fxdlx7f4wkznmghajnf0maysxpe -> 0x39d8679fd651689337e6f26aeb0a7b45fb29a5fb = 300000000000000000000000000
mezo14ugypmyfn2whx4dhsw4jwflk2x5fx2vkvmjrzk -> 0xaf1040ec899a9d7355b783ab2727f651a8932996 = 400000000000000000000000000
mezo1lgrfdvcj38gwcwr0qv4ytdfcygc0mg6slsj3yt -> 0xfa0696b31289d0ec386f032a45b5382230fda350 = 100000000000000000000000000
```

Reply:
you're right, all good then

Reply:
Nitpicking, but can we get them into the same order? I was confused for a moment just like @dimpar.

Reply:
Can we change the PR description to align with the `genesis` file? Up to you though, really not biggy.

Reply:
The description follows the order from the Linear task that holds the spec. I prefer to follow that order. The order in genesis doesn't really matter for humans.